libs/community/tests/integration_tests/vectorstores/qdrant/test_similarity_search.py

```diff
@@ -274,3 +281,18 @@ def test_qdrant_similarity_search_with_relevance_scores(
     assert all(
         (1 >= score or np.isclose(score, 1)) and score >= 0 for _, score in output
     )
+
+
+def assert_documents_equals(actual: List[Document], expected: List[Document]):
+    assert len(actual) == len(expected)
+
+    for actual_doc, expected_doc in zip(actual, expected):
+        assert actual_doc.page_content == expected_doc.page_content
+
+        assert "_id" in actual_doc.metadata
+        assert "_collection_name" in actual_doc.metadata
+
+        actual_doc.metadata.pop("_id")
+        actual_doc.metadata.pop("_collection_name")
+
+        assert actual_doc.metadata == expected_doc.metadata
```

Comment:
Can be defined at `tests/integration_tests/vectorstores/qdrant/common.py` for consistency.

Reply:
Thank you. I added additional fixes for them.
I run `pytest` command for all directories in `lib/community`, but tests for qdrant are ignored for some reason, then I missed failing tests. Sorry for the inconvenience!